applications/serial_lte_modem/Kconfig

```diff
@@ -167,6 +167,18 @@ config SLM_PPP
 config SLM_CMUX
 	bool "CMUX support in SLM"
 
+if SLM_CMUX && SLM_PPP
```

Comment:
It can, but does it make sense and what makes it a better choice?

I have a feeling that this is how Kconfig options are defined for options belonging to particular features that have a main Kconfig option. And that `depends on` is rather used for dependencies external to the feature to which the Kconfig option in question belongs.

Reply:
Fair enough.